components/automate-ui/src/app/components/time/time.component.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { StoreModule, Store } from '@ngrx/store';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { NgrxStateAtom, ngrxReducers, runtimeChecks } from 'app/ngrx.reducers';
+import { TimeComponent } from './time.component';
+import { DatetimePipe } from 'app/pipes/datetime.pipe';
+import { UserPreferencesService } from 'app/services/user-preferences/user-preferences.service';
+
+describe('TimeComponent', () => {
+  let component: TimeComponent;
+  let fixture: ComponentFixture<TimeComponent>;
+  let store: Store<NgrxStateAtom>;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [TimeComponent, DatetimePipe],
+      providers: [UserPreferencesService],
+      imports: [StoreModule.forRoot(ngrxReducers, { runtimeChecks })],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    store = TestBed.inject(Store);
+    spyOn(store, 'dispatch').and.callThrough();
+    fixture = TestBed.createComponent(TimeComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
it will be good if you can add test cases for this component

Reply:
For time component, we used datetime pipe to convert the timeformat in template, apart from that no logic in time.component.ts file. Can we resolve this comment ?

Reply:
done